joomla/joomla-cms
# [Privacy] Clarifies confusing dashboard Status Check

### The issue
Although the code differentiates the result of `Mail Sending` by providing different language strings (`Mail Sending Enabled` & `Mail Sending Disabled`) and proposing a tip when disabled, it does not take care of the real status of the `Privacy Policy article` and the `Request Form Menu Item`.

The buttons were normalized, but not the strings, which indicates (I was told by @mbabker  it was for a11y reasons) the term `Published` for both items.
```
COM_PRIVACY_STATUS_CHECK_REQUEST_FORM_MENU_ITEM_PUBLISHED="Request Form Menu Item Published"
COM_PRIVACY_STATUS_CHECK_PRIVACY_POLICY_PUBLISHED="Privacy Policy Published"
```
When translating these I found there a confusion to have an `Unpublished` button and a string containing the term `Published`.

I also found out that we also have a confusion between the real `Unpublished `state of these items versus their existence itself.

### Summary of Changes
Differentiating between:
1. an unpublished Privacy Policy article when it exists (i.e defined in the Consent system plugin parameter AND in database, as it could have been deleted —while still present in the plugin parameters— or unpublished ) 
2. a Request Form Menu Item when it exists or is unpublished.

I did not change the buttons values but modified the strings of the Checked items to display their real status between parenthesis. Therefore a11y would still be OK if that was the reason for adding `Published` in the first place. 
If it was not the reason, then these can be taken off and we will absolutely need a new text for the button when the items do not exist in the database. See below.

### Testing Instructions
Create, publish, unpublish and delete a `Privacy Policy article` (created through the System Plugin - Consent.)
Create, publish and unpublish and delete a  `Request Form Menu Item` in a menu.

For each case display the /administrator/index.php?option=com_privacy dashboard and check the results.


### After patch
1. If one of these items does not exist at all in database, the button will display `Not available`

2. If one of these items exists in database BUT is unpublished in their manager, the Button will display "Unpublished"

3. If one of these items exists in database and is published the button will display "Published"

Example of results (AFTER MODIFICATIONS, SEE BELOW):
<img width="756" alt="screen shot 2018-09-22 at 18 15 07" src="https://user-images.githubusercontent.com/869724/45919358-60024980-be94-11e8-990b-601e008ef529.png">


@mbabker
